index.js

```diff
@@ -4,6 +4,8 @@ let {
   emrun_printErr,
 } = window;
 
+const autoRunEnabled = location.search.toLowerCase().indexOf('autorun') != -1;
```

Comment:
I don't disagree but since all the checks for the existence of certain elements use the `!= -1` check I'm going to defer that clean up task to a future PR 😅

Reply:
Actually, because JavaScript has no type safety, I tend to err on the side of assigning values to variables that will clearly be of a particular type. Simply invoking a function is ambiguous. If I intend a variable to be a boolean, I use expressions that will definitely result in a boolean (such as a boolean operator).

Also, to avoid JavaScript side effect fun, we should prefer !== to != whenever possible.

But none of these are issues that need to be fixed in this PR. Just stuff to consider going forward.

Reply:
I was just reading up more on this and it appears String.prototype.includes is now in the JavaScript standard. My stance is more around things that come from libraries and custom code, so I agree with David, and it would probably be good to normalize around an includes check throughout the file. Another good pass would be to replace == and != with === and !==.

Reply:
Glad you've come around to my way of thinking, Matt. ;-)